__tests__/user/user.mutation.test.js

```diff
@@ -439,7 +438,6 @@ describe('User`s mutation restictions tests', () => {
     lastName = 'Markelo';
     email = '1xamp31d2v1@gmail.com';
     password = 'qwertY123';
```

Comment:
please move constants to the variables file

Reply:
fixed